recipe/meta.yaml

```
@@ -17,12 +17,15 @@ package:
 
 source:
   url: https://boostorg.jfrog.io/artifactory/main/release/{{ version }}/source/boost_{{ version|replace(".", "_") }}.tar.bz2
-  sha256: a6e1ab9b0860e6a2881dd7b21fe9f737a095e5f33a3a874afc6a345228597ee6
+  sha256: 6478edfe2f3305127cffe8caf73ea0176c53769f4bf1585be237eb30798c3b8e
   patches:
-    - 5eff1ecc8413b0dc93a1ab047d7fed751e6cb40e.patch
+    # ensure our compiler flags get used during bootstrapping 
+    - patches/0001-Add-default-value-for-cxx-and-cxxflags-options-for-t.patch
+    # backport https://github.com/boostorg/test/commit/cada8c11df0ee360c2a05b0f44daeae6893419e4
```

Probably also worth noting that this is just relevant for old glibc's (though we juuuuust won't be able to drop it even with the move to 2.17 next year).
```suggestion
    # backport https://github.com/boostorg/test/commit/cada8c11df0ee360c2a05b0f44daeae6893419e4
    # for compatibility with old glibc (<2.18)
```